sci-mathematics/rstudio/files/rstudio-1.4.1106-add-missing-boost-bind-import.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/src/cpp/core/HtmlUtils.cpp b/src/cpp/core/HtmlUtils.cpp
```

Comment:
Is any of this upstreamable btw?

Reply:
Very good question - probably yes, though the codebase is so vast that getting involved with upstream is a tad daunting.

Perhaps once I get the ebuild buttoned up i can work on a single megaissue for upstream.

Reply:
Sure.

Reply:
👍 I'll close this as resolved and add a task to the `TODO` list at the top of the PR